### ISSUE TITLE
Update Readme.md about diskmgmt does not show info about BTRFS volumes

### ISSUE THREAD

**Issue by SecriusV** [maharmstone/btrfs#715]
I was spend about 2 hours to switch this driver on, and to gather all debug information about btrfs driver doesn't works.

After i install driver I opened diskmgmt.msc tool for check installation success, and find that not one of my btrfs volumes does not recognized.

But after i opened Explorer for attach debug info to ticket, i saw, that i have new 2 drives in "This PC".

Can you please add a **Note** to README that "Information about BTRFS filesystem volumes will not displays in diskmgmt snap-in. You cand find your BTRFS-volumes in Explorer"?

Or, the better option — add support for diskmgmt snap-in, if it is possible.

**Comment by maharmstone** [comment]
You're right, this isn't obvious. I've added a bit to the README: 75865c5d

**Comment by sskras** [comment]
Mark, have you also thought about implementing changes that would enable Virtual Disk Service (VDS) to recognize the BTRFS volume, please?

**Comment by sskras** [comment]
At least the [**Ext4FSD**](http://www.accum.se/~bosse/#:~:text=Ext4%20filesystem%20driver%20for%20Windows) project [v0.71](https://github.com/bobranten/Ext4Fsd/releases/tag/v0.71) knows how to make it show EXT4 instead of RAW currently displayed for BTRFS volumes:

<img width="991" height="831" alt="Image" src="https://github.com/user-attachments/assets/fed5f1e2-276e-4056-8ea3-3a39a6054fb6" /> <br />

FWIW, the related outputs from `fsutil fsinfo volumeinfo`:
```cmd
C:\Users\saukrs> gsudo fsutil fsInfo volumeinfo \\?\Volume{0869fa25-bd66-7074-6498-7f4d08d64db0}\
Volume Name : A Btrfs Vol
Volume Serial Number : 0x8d64db0
Max Component Length : 255
File System Name : NTFS
Is ReadWrite
Not Thinly-Provisioned
Supports Case-sensitive filenames
Preserves Case of filenames
Supports Unicode in filenames
Preserves & Enforces ACL's
Supports Sparse files
Supports Reparse Points
Supports POSIX-style Unlink and Rename
Supports Object Identifiers
Supports Named Streams
Supports Hard Links
Supports Extended Attributes
Supports Open By FileID
Supports Block Cloning
```
```cmd
C:\Users\saukrs> gsudo fsutil fsInfo volumeinfo \\?\Volume{aaa18ef8-45ce-48e0-8b2a-9896b4976d2c}\
Volume Name : An EXT4 Vol
Volume Serial Number : 0x6ff17ae3
Max Component Length : 255
File System Name : EXT4
Is ReadWrite
Not Thinly-Provisioned
Supports Case-sensitive filenames
Preserves Case of filenames
Supports Reparse Points
Supports Hard Links
Supports Extended Attributes
```

**Comment by lesderid** [comment]
@sskras See https://github.com/maharmstone/btrfs/commit/75865c5d0281331f486f15c137e7d861e5c6a079 or https://github.com/maharmstone/btrfs/issues/794#issuecomment-4669359057.

**Comment by sskras** [comment]
OK, but I'd like more technical details on that issue.
Or maybe I should set up an instance of native Storage Spaces, and compare the result to the WinBTRFS setup.

**Comment by sskras** [comment]
Well, I connected two physical drives to my setup – Disk 1 and Disk 2:

<img width="1026" height="920" alt="Image" src="https://github.com/user-attachments/assets/c1734b74-0f1e-4ef1-bf40-d13368f3dedf" /> <br />

After I added them the pool of Storage Spaces (a mechanism native to Windows 10 and similar to BTRFS):

<img width="1029" height="516" alt="Image" src="https://github.com/user-attachments/assets/678ba01e-4c7a-4ee7-9092-ee396c9f55e9" /> <br />

... it hid both of them from `diskmgmt.msc`:

<img width="885" height="920" alt="Image" src="https://github.com/user-attachments/assets/8ac0229f-6e9e-4a29-8bfd-076d0b8cb96d" /> <br />

Now if I create a space on top of the pool:

<img width="1029" height="884" alt="Image" src="https://github.com/user-attachments/assets/9c09eaad-1512-4beb-915e-945f019cc056" /> <br />

... it introduces the third, new physical disk, represented by the new `D:` volume:

<img width="1026" height="920" alt="Image" src="https://github.com/user-attachments/assets/8b769926-8cdf-4bf4-b290-df3c19684edd" /> <br />

So avoiding `RAW` here is definitely doable in the whole-disk setup (the only way the SS work).
Now only to mock up a scenario for the partition-based setup.

**Comment by maharmstone** [comment]
Because Disk Management knows about Storage Spaces! It doesn't know anything about what we're doing.
This is purely cosmetic, and I would not spend too much time worrying about it.

**Comment by sskras** [comment]
Mark, surely it is cosmetic. That's the whole point of the issue with UI :)

Anyways, I came up with the following mock-up:

<img width="1026" height="766" alt="Image" src="https://github.com/user-attachments/assets/564cf7e0-75d6-49ec-bfbb-8b9b9a715d2b" /> <br />

The imagined `BTRFS-vol` type of FS here is purely cosmetic – could be implemented only to put the string into a placeholder.
Meanwhile the `BTRFS` type of FS could be assigned to a newly mounted volume residing on the simulated "physical drive" (the latter actually being virtual, as in the case of SS).

Not to push anyone into doing that – just showing the UI improvement to be possible. Cheers! :)
